packages/ui/src/components/ui/legacy/calendar/calendar-matrix.tsx

```
@@ -7,6 +7,10 @@
 import isSameOrBefore from 'dayjs/plugin/isSameOrBefore';
 import timezone from 'dayjs/plugin/timezone';
 import { useParams } from 'next/navigation';
+import { useCalendar } from '../../../../hooks/use-calendar';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useCalendar'. Consider to delete it or rename it.

'useCalendar' is defined here:


(lint/suspicious/noRedeclare)